includes/class-wc-stripe-mode.php

```diff
@@ -24,6 +24,6 @@ public static function is_live() {
 	 */
 	public static function is_test() {
 		$settings = WC_Stripe_Helper::get_stripe_settings();
-		return 'yes' === $settings['testmode'];
+		return 'yes' === ( $settings['testmode'] ?? 'no' );
```

Comment:
It's a very unusual scenario, but if in any case `$settings['testmode']` is not defined the `is_live` and `is_test` both functions will return false. 

Probably `is_live` should return `true` in this scenario.

Reply:
I thought about that as well. But I did a test here, and it behaves as you suggested already:
```php
var_dump( 'yes' !== ( $settings['testmode'] ?? 'no' ) ); // `is_live()`: prints `true`
var_dump( 'yes' === ( $settings['testmode'] ?? 'no' ) ); // `is_test()`: prints `false`
```